.github/workflows/ci.yml

```
@@ -1,7 +1,7 @@
 name: CI
 
 env:
-  DENO_VERSION: 1.4.6
+  DENO_VERSION: 1.6.0
```

Let's go directly to 1.6.3 since that is the latest at the time of writing.
```suggestion
  DENO_VERSION: 1.6.3
```